build.gradle

```diff
@@ -27,11 +26,13 @@ def watchForChange = 'src/main/kotlin/**/*'
 def doOnChange = './gradlew classes'
 
 dependencies {
+  implementation "commons-lang:commons-lang:2.6"  // Required for StringEscapeUtils
```

Comment:
Vert.x's `StringEscapeUtils` has been removed since v4:
https://github.com/eclipse-vertx/vert.x/commit/ea0eddb129530ab3719c0ef86b471894876ec519

It was a clone from Apache Commons Lang (2). Here uses the original one instead of Vert.x's clone.

We may want to replace `StringEscapeUtils` with something else in the future.